newsfragments/1668.bugfix.rst

```diff
@@ -1 +1 @@
-Fix type annotations within the ``eth.py`` module. Several arguments that defaulted to ``None`` were not declared ``Optional``.
+Fix type annotations within the ``eth.py`` and ``contract.py`` modules. Several arguments that defaulted to ``None`` were not declared ``Optional``.
```

Comment:
Let's get a separate newsfragment going, just because it'll automagically link back to this PR.

Reply:
Ok!